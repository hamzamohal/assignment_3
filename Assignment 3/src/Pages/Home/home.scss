.home-title {
  margin-bottom: 2rem;
}

.home-message > p {
  font-size: 16px;
  margin-bottom: 1.5rem;
  font-size-adjust: 0.5;
  line-height: calc(1ex / 0.32);
}
