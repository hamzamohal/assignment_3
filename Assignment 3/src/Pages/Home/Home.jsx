import './home.scss';

const Home = () => (
  <div className="home">
    <h1 className="home-title">Welcome to Math Magicians</h1>
    <article className="home-message">
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit.
        Unde nemo libero ipsa beatae! Architecto eos, tempore, eveniet velit eius totam provident
        voluptates aspernatur labore dolorem, facere harum quibusdam deleniti saepe.
      </p>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit.
        Unde nemo libero ipsa beatae! Architecto eos, tempore, eveniet velit eius totam provident
        voluptates aspernatur labore dolorem, facere harum quibusdam deleniti saepe.
      </p>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit.
        Unde nemo libero ipsa beatae! Architecto eos, tempore, eveniet velit eius totam provident
        voluptates aspernatur labore dolorem, facere harum quibusdam deleniti saepe.
      </p>
    </article>
  </div>
);

export default Home;
