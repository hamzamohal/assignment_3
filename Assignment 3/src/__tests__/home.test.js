import TestRenderer from 'react-test-renderer';
import Home from '../Pages/Home/Home';

describe('Test Home component', () => {
  it('should renders Home component', () => {
    const homeComponent = TestRenderer.create(<Home />).toJSON();
    expect(homeComponent).toMatchSnapshot();
  });
});
