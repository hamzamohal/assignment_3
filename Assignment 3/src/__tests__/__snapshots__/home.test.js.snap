// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Home component should renders Home component 1`] = `
<div
  className="home"
>
  <h1
    className="home-title"
  >
    Welcome to Math Magicians
  </h1>
  <article
    className="home-message"
  >
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Unde nemo libero ipsa beatae! Architecto eos, tempore, eveniet velit eius totam provident voluptates aspernatur labore dolorem, facere harum quibusdam deleniti saepe.
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Unde nemo libero ipsa beatae! Architecto eos, tempore, eveniet velit eius totam provident voluptates aspernatur labore dolorem, facere harum quibusdam deleniti saepe.
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Unde nemo libero ipsa beatae! Architecto eos, tempore, eveniet velit eius totam provident voluptates aspernatur labore dolorem, facere harum quibusdam deleniti saepe.
    </p>
  </article>
</div>
`;
